src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/tests/Datagrid.test.js

```diff
@@ -37,38 +40,48 @@ jest.mock('../../../services/Translator', () => ({
     },
 }));
 
+class TestAdapter extends AbstractAdapter {
+    static getLoadingStrategy: () => string = () => { return 'pagination'; };
+    static getStorageStrategy: () => string = () => { return 'flat'; };
+
+    render() {
+        return (
+            <div>Test Adapter</div>
+        );
+    }
+}
+
 beforeEach(() => {
     datagridAdapterRegistry.has.mockReturnValue(true);
-    datagridAdapterRegistry.get.mockReturnValue({
-        Adapter: TableAdapter,
-        paginationType: 'default',
-    });
+    datagridAdapterRegistry.get.mockReturnValue(TestAdapter);
```

Comment:
I would return a simple `jest.fn()` instead of a real implementation. That's less code, and has the advantage that calls can be asserted. Unless you tried and it was not working for some reason?